File: java/main/src/entities/pessoa.java
package entities;

public class pessoa {
    private String nome;
    private String cpf;
    private String email;
    private String telefone;

    public pessoa(String nome, String cpf, String email, String telefone) {
        this.nome = nome;
        this.cpf = cpf;
        this.email = email;
        this.telefone = telefone;
    }

    public String getNome() {
        return nome;
    }

    public String getCpf() {
        return cpf;
    }

    public String getEmail() {
        return email;
    }

    public String getTelefone() {
        return telefone;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public void setCpf(String cpf) {
        this.cpf = cpf;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public void setTelefone(String telefone) {
        this.telefone = telefone;
    }

    @Override
    public String toString() {
        return "Nome: " + nome + "\nCPF: " + cpf + "\nEmail: " + email + "\nTelefone: " + telefone;
    }

}

File: labs/poo/lab07/Pessoa.cpp
#include "Pessoa.hpp"

Pessoa::Pessoa(const std::string &nome, const std::string &cpf)
    : nome(nome), cpf(cpf) {}

std::string Pessoa::getNome() const
{
    return nome;
}

std::string Pessoa::getCpf() const
{
    return cpf;
}

void Pessoa::setNome(const std::string &nome)
{
    this->nome = nome;
}

void Pessoa::setCpf(const std::string &cpf)
{
    this->cpf = cpf;
}